src/main/java/io/excaliburfrc/robot/subsystems/Climber.java

```diff
@@ -51,18 +51,22 @@ public ClimberSide(int motorId, boolean isMotorReversed) {
       encoder.setPosition(0);
     }
 
-    public Command pullUpCommand() {
+    public Command pullUpCommand(RelativeEncoder otherEncoder) {
       return new FunctionalCommand(
           () -> {},
-          () -> motor.set(-OPEN_LOOP_CLIMB_DUTY_CYCLE),
+          () ->
+              motor.set(
+                  -OPEN_LOOP_CLIMB_DUTY_CYCLE * otherEncoder.getPosition() / encoder.getPosition()),
```

Comment:
We want this as the delta (and not ratio) because we want the correction effort to not be based on whether the deviation happens at the top or bottom